pkg/asset/internal/templates.go

```diff
@@ -222,6 +222,13 @@ spec:
     - --authorization-mode=Node,RBAC
     - --bind-address=0.0.0.0
     - --client-ca-file=/etc/kubernetes/secrets/ca.crt
+    - --requestheader-client-ca-file=/etc/kubernetes/secrets/aggregator.crt
```

Comment:
nit : can this be renamed to `aggregation-ca` ?

Reply:
Even better, I aligned with the standard naming convention and called it `front-proxy-ca.crt`.